irohad/torii/impl/command_service_transport_grpc.hpp

```diff
@@ -106,15 +128,17 @@ namespace torii {
 
     std::shared_ptr<CommandService> command_service_;
     std::shared_ptr<iroha::torii::StatusBus> status_bus_;
-    const std::chrono::milliseconds initial_timeout_;
-    const std::chrono::milliseconds nonfinal_timeout_;
     std::shared_ptr<shared_model::interface::TxStatusFactory> status_factory_;
     std::shared_ptr<TransportFactoryType> transaction_factory_;
     std::shared_ptr<shared_model::interface::TransactionBatchParser>
         batch_parser_;
     std::shared_ptr<shared_model::interface::TransactionBatchFactory>
         batch_factory_;
     logger::Logger log_;
+
+    std::mutex round_update_mutex_;
+    std::shared_ptr<iroha::network::ConsensusGate> consensus_gate_;
+    const int kMaximumRoundsWithoutUpdate;
```

Comment:
It's a constant since you set it in a constructor, so usual naming should be used here.